e2e/install2_test.go

```diff
@@ -0,0 +1,288 @@
+package e2e
+
+import (
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/replicatedhq/embedded-cluster/e2e/cluster"
+	"github.com/replicatedhq/embedded-cluster/e2e/cluster/docker"
+	"github.com/replicatedhq/embedded-cluster/e2e/cluster/lxd"
+	"github.com/stretchr/testify/require"
+)
+
+// TODO: Remove this in favor of singleNodeInstallUpgradeTest
+func singleNodeInstallTest(t *testing.T, tc cluster.Cluster, additionalArgs []string) {
+	t.Logf("%s: installing embedded-cluster on node 0", time.Now().Format(time.RFC3339))
+	line := []string{"single-node-install2.sh", "ui", os.Getenv("SHORT_SHA"), "--admin-console-port", "30002"}
+	line = append(line, additionalArgs...)
+	if stdout, stderr, err := tc.RunCommandOnNode(0, line); err != nil {
+		t.Fatalf("fail to install embedded-cluster on node 0: %v: %s: %s", err, stdout, stderr)
+	}
+
+	if stdout, stderr, err := tc.SetupPlaywrightAndRunTest("deploy-app"); err != nil {
+		t.Fatalf("fail to run playwright test deploy-app: %v: %s: %s", err, stdout, stderr)
+	}
+
+	// TODO: check installation state
+}
+
+func singleNodeInstallUpgradeTest(t *testing.T, tc cluster.Cluster, additionalArgs []string) {
+	t.Logf("%s: installing embedded-cluster on node 0", time.Now().Format(time.RFC3339))
+	line := []string{"single-node-install2.sh", "ui", os.Getenv("SHORT_SHA"), "--admin-console-port", "30002"}
+	line = append(line, additionalArgs...)
+	if stdout, stderr, err := tc.RunCommandOnNode(0, line); err != nil {
+		t.Fatalf("fail to install embedded-cluster on node 0: %v: %s: %s", err, stdout, stderr)
+	}
+
+	if stdout, stderr, err := tc.SetupPlaywrightAndRunTest("deploy-app"); err != nil {
+		t.Fatalf("fail to run playwright test deploy-app: %v: %s: %s", err, stdout, stderr)
+	}
+
+	// TODO: check installation state
+
+	appUpgradeVersion := fmt.Sprintf("appver-%s-upgrade", os.Getenv("SHORT_SHA"))
+	testArgs := []string{appUpgradeVersion}
+
+	t.Logf("%s: upgrading cluster", time.Now().Format(time.RFC3339))
+	if stdout, stderr, err := tc.RunPlaywrightTest("deploy-upgrade", testArgs...); err != nil {
+		t.Fatalf("fail to run playwright test deploy-app: %v: %s: %s", err, stdout, stderr)
+	}
```

Comment:
i wouldnt call this done until youve implemented `check-postupgrade-state.sh`. Otherwise you are regressing functionality and its hidden.

Reply:
I agree that it's not done yet, but it's still better than nothing
this just closes out the 'make airgap work' ticket while I continue on with the 'test install2' ticket

Reply:
added a basic check now